packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -86,6 +92,12 @@ SBEditor class >> openFor: aMethodOrClass [
 	^ (self newFor: aMethodOrClass) openInWindow; yourself
 ]
 
+{ #category : #'initialize-release' }
+SBEditor class >> registerInWorldMenu [
+
+	TheWorldMenu registerOpenCommand: {'Sandblocks'. {self. #open}}
```

Comment:
@tom95 Or do you think we should use `#openExample` here?

Reply:
I would say a plain editor is better suited here, otherwise you get the "default cube" situation as it is in Blender :D (if you've never heard of it, it's a meme that the first thing you do when opening Blender is to delete the default cube; even if you right after create a new cube)